components/Nav.js

```diff
@@ -48,7 +51,25 @@ export default function Nav() {
           <span className={`${menu} ${isItems}`}>중고마켓</span>
         </Link>
       </div>
-      <button className={loginButton}>로그인</button>
+      {user ? (
+        <div
+          className="w-[96px] h-[40px] flex items-center justify-between mr-[200px]
+          md:w-[40px] md:mr-[24px] sm:w-[40px] sm:mr-[16px]"
+        >
+          <img
+            src={user.image ? user.image : "/images/ic_profile.png"}
```

Comment:
@taeyeonkim94 아래처럼 바꿀수있어요

```suggestion
            src={user.image ?? "/images/ic_profile.png"}
```

```suggestion
            src={user.image || "/images/ic_profile.png"}
```